.github/ISSUE_TEMPLATE/proposal.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Event/Presentation/Happening Proposal
+about: Suggest what we could do at the dat event
+title: ''
+labels: 'proposal'
+assignees: ''
+
+---
+
+**What do you suggest we do?**
+
+_Title_: {Insert title here}
+
+{Text and/or images describing what we could do.}
```

Comment:
To let people tell us what to do I tried to keep it as open as possible, using as little assumptions as possible.

Reply:
I think it's still helpful to have some guidance about what they should include in their description to prevent 1 sentence offers.

Reply:
agree with @okdistribute. We can include/mention that is important to add some basic info like: a title, description, etc. Also another thing we can do is to link to good/well known CFP entries as an example, this is a common thing between CFPs but maybe due to our novel approach to a diverse CFP (not only talks) this could be challenging but any good example can help.